js/src/components/schema/ColumnNameCell.tsx

```diff
@@ -105,6 +110,13 @@ export function ColumnNameCell({
                       disabled={addedOrRemoved}>
                       Top-k Diff
                     </Menu.Item>
+                    <Menu.Item
+                      value="value-diff"
+                      fontSize="10pt"
```

Comment:
don't use "pt" -> either use a chakra-defined size or use `rem` -> 10pt is about `0.85rem`

Reply:
The other menu item `fontSize` fields should be updated as well - i can see this was an existing issue